src/module/item/affliction/document.ts

```diff
@@ -4,17 +4,11 @@ import { UserPF2e } from "@module/user/index.ts";
 import { createDamageFormula, parseTermsFromSimpleFormula } from "@system/damage/formula.ts";
 import { AfflictionDamageTemplate, BaseDamageData, DamagePF2e, DamageRollContext } from "@system/damage/index.ts";
 import { DamageRoll } from "@system/damage/roll.ts";
-import { ErrorPF2e } from "@util";
+import { ItemSummaryData } from "@item/data/index.ts";
 import { AfflictionFlags, AfflictionSource, AfflictionSystemData } from "./data.ts";
+import { DegreeOfSuccess } from "@system/degree-of-success.ts";
 
 class AfflictionPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends AbstractEffectPF2e<TParent> {
-    constructor(source: object, context?: DocumentConstructionContext<TParent>) {
-        super(source, context);
-        if (BUILD_MODE === "production") {
-            throw ErrorPF2e("Affliction items are not available in production builds");
-        }
-    }
-
```

Comment:
shark isn't gonna be too happy about this one, though I personally think showing a ui notification error is fine instead. You'll have to ask him for this one :(

Reply:
@stwlam Is it ok if I delete this one check? I can change error to ui notification.

Reply:
I'm guessing not. The rest of this looks fine, and I pushed up some condition granting stuff elsewhere. Lets at least try to finish this as soon as possible. If we're fast enough it can be either the next release or the one after.

Reply:
So I will roll back this change